### PR TITLE
Unable to display custom error thrown from the changePassword action

### PR DESCRIPTION
**Pull request by manojleaton**
Fixes #BLUI-5969 .

#### Changes proposed in this Pull Request:

- Fix for the [issue](https://github.com/etn-ccis/blui-react-workflows/issues/612)

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

- https://github.com/etn-ccis/blui-react-workflows/blob/5270d35d2f251e3337f92f387a13d2fbf6098702/login-workflow/example/src/actions/AuthUIActions.tsx#L172
- return true to display custom error  pass 

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Comment by JeffGreiner-eaton** [comment]
same issues as before, react and types-react versions throwing warnings...
image

**Review by JeffGreiner-eaton — APPROVED** [review]

**Comment by ektaghag-eaton** [comment]
Screenshot 2024-08-08 at 1 07 26 PM

it is working

**Review by ektaghag-eaton — APPROVED** [review]

**Review by ektaghag-eaton — COMMENTED** [review]
hi @manojleaton , please update changelog

File: login-workflow/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v4.0.3 (Aug 9, 2024)
+
+### Added
+
+-   Error Maneger for ChangePasswordDialog ([#612](https://github.com/etn-ccis/blui-react-workflows/issues/612))
```
Comment (ektaghag-eaton) [inline login-workflow/CHANGELOG.md:12]: please fix typo

File: login-workflow/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v4.0.3 (Aug 9, 2024)
```
Comment (JeffGreiner-eaton) [inline login-workflow/CHANGELOG.md:8]: you can use 4.0.4 or the next release version here with no date (unreleased) 4.0.3 was released in may

**Pull request merged**